mysql-test/suite/encryption/t/doublewrite_debug.test

```diff
@@ -0,0 +1,223 @@
+--source include/have_innodb.inc
+--source include/have_debug.inc
+--source include/not_embedded.inc
+--source include/have_example_key_management_plugin.inc
+call mtr.add_suppression("InnoDB: Encrypted page \\[page id: space=[1-9][0-9]*, page number=3\\] in file .*test.t[12].ibd looks corrupted");
```

Comment:
Actually it should be `\\.ibd` instead of `.ibd` because we are looking for a literal `.` in the file name, instead of just any character. But, I don’t feel strongly about this little bit of sloppiness.